hazelcast/src/main/java/com/hazelcast/instance/impl/BootstrappedJetProxy.java

```diff
@@ -61,8 +61,10 @@ protected BootstrappedJetProxy(@Nonnull JetService jetService) {
 
     public abstract boolean hasExecuteJobParameters();
 
+    // The strategy to get ExecuteJobParameters on client and member side
```

Comment:
Please change it to JavaDoc

Reply:
rev : 35bc7755ad47b74ac4e7f21e3a3e9a0fa9f37144